Plugins/Public/base_plugin/ClientCommands.cpp

```diff
@@ -156,51 +156,53 @@ void SendBaseStatus(uint client, PlayerBase* base)
 	const Universe::ISystem* sys = Universe::get_system(base->system);
 
 	wstring base_status = L"<RDL><PUSH/>";
-	base_status += L"<TEXT>" + XMLText(base->basename) + L", " + HkGetWStringFromIDS(sys->strid_name) + L"</TEXT><PARA/><PARA/>";
+	base_status += L"<TEXT>" + XMLText(base->basename) + L", " + HkGetWStringFromIDS(sys->strid_name) + L"</TEXT><PARA/>";
+	
+	wstring affiliation_string = L"";
+	if (base->affiliation)
+	{
+		affiliation_string = HkGetWStringFromIDS(Reputation::get_name(base->affiliation));
+	}
+	else
+	{
+		affiliation_string = L"Unaffiliated";
+	}
+
+	base_status += L"<TEXT>Core " + IntToStr(base->base_level) + L" " + affiliation_string + L" Installation</TEXT><PARA/><PARA/>";
 
 	base_status += base->infocard;
 
-	base_status += L"<TEXT>Base Core: Level " + IntToStr(base->base_level) + L"</TEXT><PARA/>";
-	base_status += L"<TEXT>Free Cargo Storage: " + Int64ToPrettyStr(base->GetRemainingCargoSpace()) + L"</TEXT><PARA/>";
-	base_status += L"<TEXT>Max Cargo Storage: " + Int64ToPrettyStr(base->GetMaxCargoSpace()) + L"</TEXT><PARA/>";
+	base_status += L"<TEXT>Cargo Storage: " + Int64ToPrettyStr(base->GetRemainingCargoSpace()) + L" free of " + Int64ToPrettyStr(base->GetMaxCargoSpace()) + L"</TEXT><PARA/>";
 	base_status += L"<TEXT>Money: " + Int64ToPrettyStr(base->money) + L"</TEXT><PARA/>";
 	if (!base->invulnerable)
 	{
-		base_status += L"<TEXT>Hit Points: " + Int64ToPrettyStr((INT64)base->base_health) + L"</TEXT><PARA/>";
+		wstring max_hp_string = L"";
```

Comment:
Just leave uninitialized and assign it on lines 182 and 186 instead of concatenating to an empty one.